### PR TITLE
landing page

### PR DESCRIPTION
**Pull request by ilhom2497**
created landing page
Screenshot 2024-07-26 001934

### CODE REVIEW

File: src/ilkhom/ilkhom.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Poppins:wght@300;400;500;600&display=swap');
+
+body {
+  color: #2958e5;
+}
+
+h1 {
+  font-size: 4em;
```
Comment (zheksha) [inline src/ilkhom/ilkhom.css:8]: Let's use `rem`

File: src/ilkhom/ilkhom.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import './ilkhom.css'
+import truckImage from './images/ffa5726971b2da124f5b1e378814ef75.png'
+import companies from './images/companies.jpeg'
+import jobs from './images/jobs.jpeg'
+import trucks from './images/trucks.jpeg'
+import mechanics from './images/mechanics.jpeg'
+import trafficLawyers from './images/traffic-lawyers.jpeg'
+import other from './images/other.jpeg'
+
+const Ilkhom = () => {
```
Comment (zheksha) [inline src/ilkhom/ilkhom.jsx:11]: Rename the component to actual name of the component

File: src/ilkhom/ilkhom.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+              <div className="mt-3">
+                <p>Other...</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  )
+}
```
Comment (zheksha) [inline src/ilkhom/ilkhom.jsx:96]: Whole component is big, hard to read, please segregate into different sections, if possible create another components for the images in for of link. Basically it is going to be Link in the future.

File: src/ilkhom/ilkhom.css (hunk 1 above)
Comment (ilhom2497) [reply src/ilkhom/ilkhom.css:8]: used only rem o

File: src/ilkhom/ilkhom.jsx (hunk 3 above)
Comment (ilhom2497) [reply src/ilkhom/ilkhom.jsx:96]: created separate components

**Pull request closed without merging**